workflows/gisaid.wdl

```diff
@@ -0,0 +1,275 @@
+version 1.1
+
+workflow LoadGISAID {
+    input {
+        String docker_image_id = "aspen-gisaid"
+        String aws_region = "us-west-2"
+        String db_data_bucket = "aspen-db-data-dev"
+        String gisaid_ndjson_export_url = "https://www.epicov.org/epi3/3p/exp3/export/export.json.bz2"
+        String gisaid_ndjson_staging_bucket = "aspen-data-dev"
+        String gisaid_ndjson_staging_key = "raw_gisaid_dump/test.zst"
```

Comment:
why is this test.xst?  also, this implies that every run will write to the same place.

Reply:
Also a test placeholder, expected to be overridden at runtime.

Reply:
In general I agree these placeholder defaults should be removed, and test values should be set in the test suite instead.

However in the particular case of gisaid dump caching (see the RefreshGISAID task - the idea is not to download and decompress the large upstream file every time if it has not changed), if this input changes, that task will need a separate pointer to the most recently downloaded copy of the file as well, in order for the caching logic to work.